content/blog/2023/06/16/2023-06-16-jenkins-may-newsletter.adoc

```diff
@@ -0,0 +1,174 @@
+
+* link:/blog/2023/05/26/ash-sxn-introduction-blog-post/[Ashutosh Saxena]
+* link:/blog/2023/05/22/jagruti-introduction-blog-post/[Jagruti Tiwari]
+* link:/blog/2023/05/18/harsh-ps-2003-introduction-blog-post/[Harsh Pratap Singh]
+* link:/blog/2023/05/17/vandit1604-introduction-blog-post/[Vandit Singh]
+
+We're very pleased that these new Jenkins contributors have seen the value of documentation and have submitted documentation improvements.
+
+We also welcomed first time documentation contributors in May and are pleased to have their additions.
+Thanks so much to our documentation contributors.
+
+[[outreach]]
+image:/images/post-images/2023/01/12/jenkins-newsletter/outreach-and-advocacy.png[Outreach and advocacy Update]
+
+Contributed by: author:alyssat[Alyssa Tong]
+
+image:/images/post-images/2023/06/16/2023-06-16-jenkins-may-newsletter/image2.png[image,width=839,Jenkins Contributor Awards 2023]
+
+Every year the community nominates, and votes for three outstanding difference makers in the Jenkins community: Most Valuable Advocate, Security MVP, and Most Valuable Contributor. Congratulations go to:
+
+* author:daniel-beck[Daniel Beck] - Security MVP.
+* author:janfaracik[Jan Faracik] - Most Valuable Contributor.
+* author:markewaite[Mark Waite] - Most Valuable Advocate.
+
+Read what makes them link:/blog/2023/05/16/jenkins-2023-award-winners/[outstanding contributors].
+
+Many THANKS and congratulations to all award nominees!
+
+image:/images/post-images/2023/06/16/2023-06-16-jenkins-may-newsletter/image1.jpg[image,width=839,cdCon + GitOpsCon 2023]
+
+The link:https://cd.foundation/[Continuous Delivery Foundation (CDF)] hosted its fourth flagship event, cdCon, on May 8–9, 2023 in Vancouver, Canada as link:https://events.linuxfoundation.org/cdcon-gitopscon/[cdCon + GitOpsCon], co-organized with the link:https://cncf.io/[Cloud Native Computing Foundation (CNCF)].
+
+Sessions from the most widely used CI/CD and GitOps technologies including the Jenkins community were there with project updates along with various talks from community members and users.
+
+In case you missed it, below are the recorded Jenkins sessions at cdCon:
+
+* link:https://youtu.be/9o5suZ0lRss[Fidelity’s Software Delivery Platform - Frictionless Approach to Achieve Autonomic DevOps & Enhanced Security/Compliance Practices] - Jamie Plower & Evan Elms, Fidelity Investments.
+* link:https://youtu.be/Yr3zNnbggfE[CI/CD for Data Building Dev/Test Data Environments with Open Source Stacks] - Vinodhini Duraisamy, Treeverse.
+* link:https://youtu.be/9QBORPTbkaY[Intentional and Unintentional Compromises in Test Automation] - Mark Waite, Jenkins.
+* link:https://youtu.be/vz7_rAQz_9c[The Graduated Panel] https://cdcongitopscon2023.sched.com/?iframe=yes&w=100%&sidebar=yes&bg=no#[- Dan Garfield, Codefresh; Priyanka Ravi, Weaveworks; Mark Waite, CloudBees; Andrea Frittoli, IBM & Moderated by Lori Lorusso, JFrog].
```

Comment:
```suggestion
The link:https://cd.foundation/[Continuous Delivery Foundation (CDF)] hosted its fourth flagship event, cdCon, on May 8 – 9, 2023 in Vancouver, Canada as link:https://events.linuxfoundation.org/cdcon-gitopscon/[cdCon + GitOpsCon], co-organized with the link:https://cncf.io/[Cloud Native Computing Foundation (CNCF)].

Sessions from the most widely used CI/CD and GitOps technologies, including the Jenkins community, were there with project updates along with various talks from community members and users.

In case you missed it, below are the recorded Jenkins sessions at cdCon:

* link:https://youtu.be/9o5suZ0lRss[Fidelity’s Software Delivery Platform - Frictionless Approach to Achieve Autonomic DevOps & Enhanced Security/Compliance Practices] - Jamie Plower & Evan Elms, Fidelity Investments.
* link:https://youtu.be/Yr3zNnbggfE[CI/CD for Data Building Dev/Test Data Environments with Open Source Stacks] - Vinodhini Duraisamy, Treeverse.
* link:https://youtu.be/9QBORPTbkaY[Intentional and Unintentional Compromises in Test Automation] - Mark Waite, Jenkins.
* link:https://youtu.be/vz7_rAQz_9c[The Graduated Panel] https://cdcongitopscon2023.sched.com/?iframe=yes&w=100%&sidebar=yes&bg=no#[- Dan Garfield, Codefresh; Priyanka Ravi, Weaveworks; Mark Waite, CloudBees; Andrea Frittoli, IBM & Moderated by Lori Lorusso, JFrog].
```
small formatting and punctuation suggestion